src/domains/api/get-following.js

```diff
@@ -0,0 +1,51 @@
+import { getXataClient } from "../xata";
+
+const xata = getXataClient();
+
+export default async function ({ followerId, sort, search }) {
+  const params = { pagination: { size: 50 } };
+
+  if (sort === "Inactive") {
+    return await xata.db.accounts
+      // .filter({ followed_by: followerId })
+      .sort("calculated_metrics.average_tweets_per_year", "asc")
+      .sort("calculated_metrics.years_on_twitter", "desc")
+      .getPaginated(params);
+  } else if (sort === "Overactive") {
+    return await xata.db.accounts
+      // .filter({ followed_by: followerId })
+      .sort("calculated_metrics.average_tweets_per_year", "desc")
+      .sort("calculated_metrics.years_on_twitter", "asc")
+      .getPaginated(params);
+  } else if (sort === "Unpopular") {
+    return await xata.db.accounts
+      // .filter({ followed_by: followerId })
+      .sort("public_metrics.followers_count", "asc")
+      .getPaginated(params);
+  } else if (sort === "Overpopular") {
+    return await xata.db.accounts
+      // .filter({ followed_by: followerId })
+      .sort("public_metrics.followers_count", "desc")
+      .getPaginated(params);
+  } else if (search) {
+    return await xata.search.all(search, {
```

Comment:
Thinking we'll work on hooking this feature up to a search field on the front end.